frame_test.go

```diff
@@ -307,3 +307,36 @@ func TestFrame_RowLabelValidation(t *testing.T) {
 	}
 
 }
+
+// Ensure frame can open and retrieve a view.
+func TestFrame_DeleteView(t *testing.T) {
+	f := test.MustOpenFrame()
+	defer f.Close()
+
+	viewName := pilosa.ViewStandard + "_v"
+
+	// Create view.
+	view, err := f.CreateViewIfNotExists(viewName)
+	if err != nil {
+		t.Fatal(err)
+	} else if view == nil {
+		t.Fatal("expected view")
+	}
+
+	err = f.DeleteView("v")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if f.View(viewName) != nil {
+		t.Fatal("view still exists in frame")
+	}
+
+	// // Retrieve existing view.
```

Comment:
I don't understand what this portion of the test is checking. Can you clarify in the comment?